_episodes/02-enter-sandman.md

```diff
@@ -143,14 +141,22 @@ fi
 ~~~
 {: .language-bash}
 
-and then make it executable `chmod +x bash_exit.sh`. Now, try running it with `./bash_exit.sh hello` and `./bash_exit.sh goodbye` and see what those exit codes are.
-
+and then make it executable `chmod +x bash_exit.sh`. Now, try running it with `./bash_exit.sh hello` and `./bash_exit.sh goodbye`.
+~~~
```

Comment:
why not just `echo $?` instead of this more complicated snippet?

Reply:
I see what you mean. I was trying to have it follow the example before. Would you rather I change it 'echo $?'? 
So something like
code block:
...
and then make it executable 'chmod +bash_exit.sh'. Let's check the exit code wit `echo $?`.

Reply:
I believe if we do this we can skip/remove the part where we use the python interpreter.

Reply:
Checking the status from within python was an important learning point of the previous section, but there's no reason to use it here when we're just playing around in bash